gui/wxpython/lmgr/frame.py

```diff
@@ -149,7 +149,9 @@ def show_menu_errors(messages):
         # create widgets
         self._createMenuBar()
         self.statusbar = self.CreateStatusBar(number=1)
-        self.notebook = self._createNoteBook()
+        notebook = self._createNotebook()
+        self._createWidgets(notebook)
```

Comment:
Unless createWidgets is used also elesewhere, I think it's unnecessary function, I would just put there its content directly, it will be more readable.